src/lang/english.txt

```diff
@@ -2630,6 +2630,7 @@ STR_STATION_BUILD_COVERAGE_AREA_OFF_TOOLTIP                     :{BLACK}Don't hi
 STR_STATION_BUILD_COVERAGE_AREA_ON_TOOLTIP                      :{BLACK}Highlight coverage area of proposed site
 STR_STATION_BUILD_ACCEPTS_CARGO                                 :{BLACK}Accepts: {GOLD}{CARGO_LIST}
 STR_STATION_BUILD_SUPPLIES_CARGO                                :{BLACK}Supplies: {GOLD}{CARGO_LIST}
+STR_STATION_BUILD_INFRASTRUCTURE_COST                           :{BLACK}Maintainance cost: {GOLD}{CURRENCY_SHORT}/yr
```

Comment:
Why use `{CURRENCY_SHORT}` here instead of `{CURRENCY_LONG}`?

Reply:
I don't think players need more than six significant figures.